src/components/Profile.js

```diff
@@ -1,28 +1,20 @@
-import { useEffect, useState } from "react";
-import { fetchUserData } from "../api/api";
+import useFetchData from "../hooks/useFetchData";
+import convertKeyToCamelCase from "../utils/convertKeyToCamelCase";
 
 function Profile() {
-  const [userData, setUserData] = useState("");
-
-  const handleLoad = async () => {
-    const user = await fetchUserData();
-    setUserData(user);
-  };
-
-  useEffect(() => {
-    handleLoad();
-  }, []);
+  const userData = useFetchData("users/1")?.data;
+  const userInfo = userData ? convertKeyToCamelCase(userData[0]) : null;
```

Comment:
```suggestion
const result = userData?.[0]?.convertKeyToCamelCase() 
```
이런식으로 자바스크립트 최신문법으로도 표현이 가능합니다!